styles/css/core/variables.css

```diff
@@ -70,26 +78,25 @@
   --pgn-typography-font-size-sm: .875rem;
   --pgn-typography-font-size-lg: 1.4063rem;
   --pgn-typography-font-size-base: 1.125rem;
-  --pgn-typography-font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono",  "Courier New", monospace;
+  --pgn-typography-font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas, '"Liberation Mono"', '"Courier New"', monospace;
```

Comment:
[clarification] Is having the already quoted font names now wrapped with single quotes, too intended/needed/work as expected?

Reply:
Resolved in https://github.com/openedx/paragon/pull/3207